processor/tailsamplingprocessor/processor_decisions_test.go

```diff
@@ -5,6 +5,7 @@ package tailsamplingprocessor
 
 import (
 	"context"
+	"github.com/stretchr/testify/assert"
```

Comment:
This will likely not pass the linter, I'd recommend running `make lint` to catch formatting issues before the CI does it.